### DISCUSSION TITLE
Kotlin Support will be removed

### DISCUSSION THREAD

**Discussion by juancgalvis** [bancolombia/scaffold-clean-architecture#442] [Announcements]
In order to improve project maintainability we are deciding to remove the kotlin support, this decision is based on collected usage data of different offered gradle tasks.

Do you think we should keep this language?

**Comment by stratos-code** [comment]
Hey! Actually, I want to try kotlin with this plugin. I work with Java but I prefer Kotlin because it givme a better DX than Java.

**Reply by juancgalvis** [reply]
@anpelodo Unfortunately we have made the decision to remove Kotlin templates and support for that language.

This will be closed with issue #452
